ui/lib/core/addon/components/policy-example.hbs

```diff
@@ -32,7 +32,10 @@
       Endpoint Governing Policies (EGPs) are tied to particular paths (e.g.
       <code class="tag is-marginless is-paddingless">aws/creds/</code>
       ) instead of tokens. They use
-      <Hds::Link::Inline @href="https://docs.hashicorp.com/sentinel/language">Sentinel</Hds::Link::Inline>
+      <Hds::Link::Inline
+        @isHrefExternal={{true}}
+        @href="https://docs.hashicorp.com/sentinel/language"
+      >Sentinel</Hds::Link::Inline>
       as a language to access
       <Hds::Link::Inline @href={{doc-link "/vault/docs/enterprise/sentinel/properties"}}>properties</Hds::Link::Inline>
```

Comment:
I beeline this needs one too. Then we're all set! "External" refers to any link outside the app so unless we're routing to another page inside the app, we should include this arg.

Reply:
Ah gotcha! I just applied it to the links that were previously using the `<ExternalLink/>` component, I'll update it. Thank you!!

Reply:
That makes sense! The multiple use of "external" is confusing here. For additional context, we used to use this [DocLink](https://github.com/hashicorp/vault/blob/main/ui/lib/core/addon/components/doc-link.js#L22) component to render links to our documentation, which extended from [ExternalLink](https://github.com/hashicorp/vault/blob/main/ui/lib/core/addon/components/external-link.js#L23) and just provided the first part of the path. 

As we started using the HDS link components, we made the `doc-link` helper to use instead, which can be used directly in templates to build a doc link.